docs/administration/index.md

```diff
@@ -3,7 +3,7 @@
 - [Crud Controller](./crud-controller/index.md)
 - [Administration Menu](./administration-menu.md)
 - [Datagrid](./datagrid/index.md)
-- [Grid (Legacy)](./legacy-grid/index.md)
+- [Grid (Internal)](./legacy-grid/index.md)
```

Comment:
the folder deserves renaming too :innocent:

Reply:
Done